ghc-lib/new-instructions.md

```diff
@@ -0,0 +1,59 @@
+
+## Updated instructions for working on `ghc-lib`
+
+Here are instructions for when working on DAML surface syntax, as implemented in the the digital-assert fork of `ghc`. (linked in via `ghc-lib`)
+
+These instructions were written in November 2020, and are intended to replace the [old instructions](working-on-ghc-lib.md), but there might still be useful stuff there.
+
```

Comment:
I’d just remove this whole section. We shouldn’t merge this without ripping out the old stuff.